src/stories/contextMenuAndSelection/ContextMenuAndSelection.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import { createTestids } from '@famiprog-foundation/tests-are-demo';
+import { ComponentStory } from '@storybook/react';
+import { Alert } from 'antd';
+import React, { Fragment, useState } from 'react';
+import Timeline from '../../timeline';
+import { IGanttAction, IGanttOnContextMenuShowParam, Item } from '../../types';
+import { d, someHumanResources, someTasks } from '../sampleData';
+import { contextMenuScenarios, selectionScenarios } from './ContextMenuAndSelectionScenarios';
+import { Button, Checkbox, Icon, Menu } from 'semantic-ui-react';
+import { IActionParamForRun } from '../../components/ContextMenu/IAction';
+
+export default {
+    title: 'Features/Context Menu And Selection',
+    includeStories: /^[A-Z]/
+};
+
+export const ContextMenu = () => {
+        const [tasks, setTasks] = useState<Item[]>([...someTasks]);
+        return (
+            <Fragment>
+                <Alert message={<><b>Add task</b> action is provided only for empty selection. It also has a different label depending on which row is displayed</>}/>
+                <div style={{ display: 'flex', height: '400px' }}>
+                    <Timeline startDate={d('2018-09-20')} endDate={d('2018-09-21')} groups={someHumanResources} items={tasks}
+                        onContextMenuShow={(contextMenuShowParam: IGanttOnContextMenuShowParam) => {
+                            const actions: IGanttAction[] = [
+                                {
+                                    icon: "edit",
+                                    label: "Edit",
+                                    isVisible: (param) => { return param.selection.length == 1; },
+                                    run: (param) => {
+                                        param.closeContextMenu();
```

Comment:
nu e bun defaultul asta. In vechiul gantt nu cred ca era asa. By default, se inchide context menu automat la sfarsitul actiunii. Trebuia sa faci opt-in ca sa NU se inchida meniul si sa ii apelezi explicit inchiderea. Trebuie `IActionParamForRun.dontCloseContextMenuAfterRunAutomatically: boolean`. Doar daca userul o pune la true, atunci e obligat sa apeleze si closeContextMenu(). Putem exemplifica aici acest comportament

Reply:
ok. am modificat